.github/workflows/CI-audit.yml

```diff
@@ -8,32 +8,61 @@ on:
   workflow_call:
   workflow_dispatch:
 
-env:
-  IMAGE_NAME: zencash/sc-ci-base
-  IMAGE_TAG: noble_rust-stable_latest
-  DOCKER_COMPOSE_CMD: "docker compose -f ${GITHUB_WORKSPACE}/ci/docker-compose.yml run --rm cargo-container"
-  DOCKER_BUILD_DIR: /build
-  DOCKER_CARGO_HOME: /tmp/.cargo
-  CARGO_BINARIES_INSTALL: "cargo-audit"
-
 jobs:
   audit:
-    runs-on: ubuntu-latest
+    runs-on: warp-ubuntu-latest-x64-2x
```

Comment:
That's a really fast job. Maybe can still run on github. What about to have 2 variables: one for fast job and other for heavy jobs?

Reply:
We are running this on SMALLER instance anyway and paying 75% less then if we would run on GH Actions default runner.